controllers/profiles/getProfileByName.js

```diff
@@ -34,23 +34,27 @@ module.exports = async (req, res) => {
         status: 'error'
       });
 
+    const targetUserId = user?.dataValues?.user_id;
+
     const getFollowerCountQuery = `SELECT COUNT(user_follow_user.user_id_follower) as count_follower from user_follow_user WHERE user_id_followed = :user_id`;
     const getFollowingCountQuery = `SELECT COUNT(user_follow_user.user_id_followed) as count_following from user_follow_user WHERE user_id_follower = :user_id`;
     const isFollowingQuery = `SELECT * FROM user_follow_user WHERE user_id_followed= :user_id_followed AND user_id_follower= :user_id_follower`;
 
     const getFollowerCount = await sequelize.query(getFollowerCountQuery, {
-      replacements: {user_id: req.userId}
+      replacements: {user_id: targetUserId}
     });
 
     const getFollowingCount = await sequelize.query(getFollowingCountQuery, {
-      replacements: {user_id: req.userId}
+      replacements: {user_id: targetUserId}
     });
 
     const isFollowing = await sequelize.query(isFollowingQuery, {
-      replacements: {user_id_followed: user?.dataValues?.user_id, user_id_follower: req.userId}
+      replacements: {user_id_followed: targetUserId, user_id_follower: req.userId}
     });
 
     const getFollowerCountResult = getFollowerCount?.[0]?.[0]?.count_follower;
+    console.log('getFollowerCountResult');
+    console.log(getFollowerCountResult);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Console logging in production code is generally not a good practice as it can clutter the console output and potentially expose sensitive information. Consider removing these lines or replacing them with a proper logging solution.
```diff
-    console.log('getFollowerCountResult');
-    console.log(getFollowerCountResult);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->